packages/ariakit/src/heading/__examples__/heading/index.tsx

```diff
@@ -0,0 +1,11 @@
+import { HeadingLevel } from "ariakit";
+import { Heading } from "ariakit/heading";
+import "./style.css";
+
+export default function Example() {
+  return (
+    <HeadingLevel>
+      <Heading className="heading1">Heading 1</Heading>
+    </HeadingLevel>
```

Comment:
Maybe a more thorough example? Something like this?
```suggestion
    <HeadingLevel>
      <Heading>Lacus Et Semper Turpis Massa Commodo Cum</Heading>
      <p>Torquent penatibus ipsum, nascetur cursus primis lobortis euismod nec scelerisque montes. Blandit accumsan semper fusce magna non.</p>
      <HeadingLevel>
        <Heading>Ac Nullam</Heading>
        <p>Volutpat metus id purus dignissim fusce. Tellus egestas neque montes consectetuer tincidunt ac sagittis aenean hendrerit. Adipiscing dolor. Tristique aptent arcu phasellus.</p>
      </HeadingLevel>
      <HeadingLevel>
        <Heading>Quis Placerat</Heading>
        <p>Platea justo lectus. Praesent. Et sodales pellentesque senectus convallis cum aptent cubilia suscipit nec, justo porta.</p>
      </HeadingLevel>
    </HeadingLevel>
```

Reply:
Hey @diegohaz , I added an example without `<p>`s, just to keep it clean and simple, let me know if it's okay or not, and I'll fix the tests if it's okay

Reply:
I'd still add `<p>` because it's valuable to know whether `<p>` should go inside `<HeadingLevel>`. And I'd replace `heading 1, 2, 3` with placeholder text. As I said, it's not guaranteed that these will be h1, h2, h3. In fact, `This is heading 1` will become `h2` when we start using Heading on the docs page.

Reply:
Thanks @diegohaz , got it, so now how should I go about writing tests for this, should I add `data-testid` for each heading, or use snapshot testing ?, what do u recommend ?

Reply:
I think toMatchInlineSnapshot is enough.

Reply:
done @diegohaz , take a look at the changes and let me know if I improve it